ansible_collections/arista/avd/molecule/eos_validate_state/intended/structured_configs/dc1-leaf1a.yml

```diff
@@ -513,3 +513,6 @@ virtual_source_nat_vrfs:
 metadata:
   platform: vEOS-lab
 dns_domain: dc1.local
+dps_interfaces:
```

Comment:
as discussed over phone, it may be best ro actually add a WAN router or two in the molecule rather than to inject a Dps interface on these router as it will help your other tests coming for WAN. Let me know if you want help on this